### PR TITLE
Fixing the first save attempt when starting a new game

### PR DESCRIPTION
**Pull request by MarcelHB**
## Description
See #1337.

## Checklist

- [x] Commit messages are descriptive and explain the rationale for changes
- [x] I used the same coding style as the surrounding code 
- [x] I have tested the proposed changes
- [x] I extended the documentation, if necessary
- [x] The proposed change builds also on our build bots (check after submission)

### CODE REVIEW

File: gemrb/core/SaveGameAREExtractor.cpp [hunk 1]
```diff
@@ -37,6 +37,9 @@ SaveGameAREExtractor::~SaveGameAREExtractor() {
 }
 
 int32_t SaveGameAREExtractor::copyRetainedAREs(DataStream *destStream, bool trackLocations) {
+	if (saveGame == nullptr) {
+		return 0;
```
Comment (lynxlynxlynx) [inline gemrb/core/SaveGameAREExtractor.cpp:41]:
```suggestion
		return GEM_OK;
```

**Comment by lynxlynxlynx** [comment]
You reset the EOLs to DOS-style, please amend back — it's obvious from the diff.

**Comment by MarcelHB** [comment]
Interesting, I just applied your suggestion from out of GitHub ... Wait a minute.

**Comment by lynxlynxlynx** [comment]
Oh, maybe it was the previous commit? I fixed them after the first merge. Odd.

**Comment by lynxlynxlynx** [comment]
Thanks!

**Pull request merged**